packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeLoader.ts

```diff
@@ -24,29 +18,29 @@ export interface LoadedHierarchyPart {
 export interface ReloadOptions {
   expandedNodes: TreeModelHierarchyNode[];
   collapsedNodes: TreeModelHierarchyNode[];
-  getInstanceFilter: (node: TreeModelRootNode | TreeModelHierarchyNode<ParentHierarchyNode>) => GenericInstanceFilter | undefined;
+  getInstanceFilter: (node: TreeModelRootNode | TreeModelHierarchyNode) => GenericInstanceFilter | undefined;
   buildNode: (node: TreeModelHierarchyNode) => TreeModelHierarchyNode;
   ignoreCache?: boolean;
 }
 
 /** @internal */
 export interface IHierarchyLoader {
```

Comment:
The file is `TreeLoader` and all other related pieces: `TreeActions`, `TreeModel`, `useUnifiedTreeSelection`, etc., use "Tree" rather than "Hierarchy". Maybe change the name of this to `ITreeLoader`?